lighty-modules/lighty-gnmi/lighty-gnmi-sb/src/main/java/io/lighty/gnmi/southbound/mountpoint/broker/GnmiDataBroker.java

```diff
@@ -54,7 +59,7 @@ public DOMDataTreeWriteTransaction newWriteOnlyTransaction() {
 
             @Override
             public void close() {
-
+                LOG.warn(NOT_IMPLEMENTED);
```

Comment:
this is not exactly correct, no implementation is needed
change to `debug` level and extend message

Reply:
done